### ISSUE TITLE
Display proposal ID

### ISSUE THREAD

**Issue by chillsauce** [EOSTribe/EOSVotes-UI#24]
We can currently see the title and content, but not the proposal ID. You can have two proposals with exactly the same title/content, and currently there's no way to distinguish between the two with the current UI.

**Comment by Soleone** [comment]
Yes, I just came here to post this, glad it already exists as an open issue. Seems like something that should be introduced asap so people get used to looking at and talking about the id as well.

I would recommend taking github approach and just showing first 8 characters or so but making the rest available as well.
